custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -98,25 +98,43 @@ def _state_aggregation_query(self, state_id):
             "(({} - child_health.opened_on::date)::integer >= 0) "
             "AND (child_health.closed = 0 OR (child_health.closed_on::date - {})::integer > 0)"
         ).format(end_month_string, start_month_string)
+        open_status_daily = (
+            "(child_health.opened_on::date <= {end_month_string}) "
+            "AND (child_health.closed = 0 OR child_health.closed_on::date > {end_month_string})"
+        ).format(end_month_string=end_month_string)
         alive_in_month = "(child_health.date_death IS NULL OR child_health.date_death - {} >= 0)".format(
             start_month_string
         )
+        alive_status_daily = "(child_health.date_death IS NULL OR child_health.date_death - {} >= 0)".format(
```

Comment:
can you simplifies this same as  open_status above

Reply:
Also this will consider  those as alive who got dead on the last day of the month as well. no?

Reply:
changed